src/index.html

```diff
@@ -5,8 +5,438 @@
+            <a class="nav__link" href="#tablets">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#video">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    
+    <main>
+      <div class="cards-container">
+        <div class="card" data-qa="card">
+          <img 
+            class="card__img" 
+            src="./images/imac.jpeg" 
+            alt="image_monitor"
+          >
+    
+          <h2 class="card__product-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+    
+          <p class="card__product-code">
+            Product code: 195434
+          </p>
+    
+          <div class="card__review">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__review-count">
+              Reviews: 5
+            </p>
+          </div>
+    
+          <div class="card__price">
+            <p class="card__price-text">
+              Price:
+            </p>
+            <p class="card__price-value">
+              $2,199
+            </p>
+          </div>
+    
+          <a 
+            class="card__button" 
+            data-qa="card-hover"
+            href="#"
+          >
+            BUY
+          </a>
+        </div>
+        <div class="card">
+          <img 
+            class="card__img" 
+            src="./images/imac.jpeg" 
+            alt="image_monitor"
+          >
+    
+          <h2 class="card__product-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+    
+          <p class="card__product-code">
+            Product code: 195434
+          </p>
+    
+          <div class="card__review">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <p class="card__review-count">
+              Reviews: 5
+            </p>
+          </div>
+    
+          <div class="card__price">
+            <p class="card__price-text">
+              Price:
+            </p>
+            <p class="card__price-value">
+              $2,199
+            </p>
+          </div>
+    
+          <a 
+            class="card__button" 
+            href="#"
+          >
+            BUY
+          </a>
+        </div>
+        <div class="card">
```

Comment:
it would be better to add empty line between siblings
```suggestion
        </div>
        
        <div class="card">
```